static/styles/right_sidebar.css

```diff
@@ -202,6 +202,12 @@
     margin-top: 5px;
```

Comment:
![image](https://user-images.githubusercontent.com/25124304/122147102-db428f80-ce75-11eb-98ed-21c8fda566d1.png)

Reply:
What if we just hide the right sidebar for spectators for now?  @alya doesn't think this looks great visually, and fixing that probably requires writing nice content here, and in any case with some screen widths this won't be visible.

Reply:
Pushed a commit in #18532 for this, let me know what you think.